src/components/Book.js

```diff
@@ -1,20 +1,26 @@
 import PropTypes from 'prop-types';
+import { useEffect } from 'react';
 import { useDispatch } from 'react-redux';
-import { checkStatus, removeBook } from '../redux/books/booksSlice';
+import { booksDisplay, removeBook, Bookss } from '../redux/books/booksSlice';
 import Form from './Form';
-import books from './books';
 
-const Books = () => (
-  <div>
-    {books.map((book) => (
-      <Book title={book.title} category={book.category} author={book.author} progress="64%" chapter="Chapter 17" id={book.id} key={book.id} />
-    ))}
-    <Form />
-  </div>
-);
+const Books = () => {
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(booksDisplay());
+  }, [dispatch]);
+  return (
+    <div>
+      {Bookss.map((book) => (
+        <Book title={book.title} category={book.category} author={book.author} progress="64%" chapter="Chapter 17" id={book.id} key={book.id} />
+      ))}
+      <Form />
+    </div>
+  );
+};
```

Comment:
You have well dispatched the action to add books and as pointed by the previous reviewer, connection is made to the API. However, the books rendered here are not updated because we are not reading the redux store. Kindly utilize the useSelector (`import { useSelector } from 'react-redux'`) method to get the state from the redux store and use the books in the store in the map.